packages/core/src/electron-browser/window/electron-window-service.ts

```diff
@@ -14,11 +14,14 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import * as electron from 'electron';
```

Comment:
We import stuff from `electron` (line 19) already, maybe we just update the existing import statement, or only import what we need.